lib/ui/semantics.dart

```diff
@@ -636,19 +638,24 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     Float64List transform,
     Int32List childrenInTraversalOrder,
     Int32List childrenInHitTestOrder,
-    Int32List customAcccessibilityActions,
+    Int32List additionalActions,
   ) native 'SemanticsUpdateBuilder_updateNode';
 
-  /// Update the custom accessibility action associated with the given `id`.
+  /// Update the custom semantics action associated with the given `id`.
   ///
-  /// The name of the action exposed to the user is the `label`. The text
+  /// The name of the action exposed to the user is the `label`, and the `hint`
+  /// is a semantic description of what performing the action does.The text
```

Comment:
nit: missing " ".

Should this include an example what the hint should look like? I believe unlike for SemanticsNode.hint you shouldn't say "double-tap to open calendar" because Android generates half of that.